Build failing

Getting an error (shown below)

Seems like the current version of this package: https://github.com/pocka/storybook-addon-designs

Has moved everything from `/lib` to `/esm`

Updating `panel.js` with an amended path seems to allow the build to complete.


```
WARN force closed preview build

ModuleNotFoundError: Module not found: Error: Can't resolve 'storybook-addon-designs/lib/register/components/Image' in '/Users/borgca/Repo/fox/node_modules/@dreipol/storybook-figma-addon/lib'
    at /Users/borgca/Repo/fox/node_modules/@storybook/core/node_modules/webpack/lib/Compilation.js:925:10
    at /Users/borgca/Repo/fox/node_modules/@storybook/core/node_modules/webpack/lib/NormalModuleFactory.js:401:22
    at /Users/borgca/Repo/fox/node_modules/@storybook/core/node_modules/webpack/lib/NormalModuleFactory.js:130:21
    at /Users/borgca/Repo/fox/node_modules/@storybook/core/node_modules/webpack/lib/NormalModuleFactory.js:224:22
    at /Users/borgca/Repo/fox/node_modules/neo-async/async.js:2830:7
    at /Users/borgca/Repo/fox/node_modules/neo-async/async.js:6877:13
    at /Users/borgca/Repo/fox/node_modules/@storybook/core/node_modules/webpack/lib/NormalModuleFactory.js:214:25
    at /Users/borgca/Repo/fox/node_modules/enhanced-resolve/lib/Resolver.js:213:14
    at /Users/borgca/Repo/fox/node_modules/enhanced-resolve/lib/Resolver.js:285:5
    at eval (eval at create (/Users/borgca/Repo/fox/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:13:1)
    at /Users/borgca/Repo/fox/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:44:7
    at /Users/borgca/Repo/fox/node_modules/enhanced-resolve/lib/Resolver.js:285:5
    at eval (eval at create (/Users/borgca/Repo/fox/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:13:1)
    at /Users/borgca/Repo/fox/node_modules/enhanced-resolve/lib/Resolver.js:285:5
    at eval (eval at create (/Users/borgca/Repo/fox/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:25:1)
    at /Users/borgca/Repo/fox/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:67:43
resolve 'storybook-addon-designs/lib/register/components/Image' in '/Users/borgca/Repo/fox/node_modules/@dreipol/storybook-figma-addon/lib'
  Parsed request is a module
  using description file: /Users/borgca/Repo/fox/node_modules/@dreipol/storybook-figma-addon/package.json (relative path: ./lib)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /Users/borgca/Repo/fox/node_modules/@dreipol/storybook-figma-addon/lib/node_modules doesn't exist or is not a directory
      /Users/borgca/Repo/fox/node_modules/@dreipol/node_modules doesn't exist or is not a directory
      /Users/borgca/Repo/fox/node_modules/node_modules doesn't exist or is not a directory
      /Users/borgca/Repo/node_modules doesn't exist or is not a directory
      /Users/borgca/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      looking for modules in /Users/borgca/Repo/fox/node_modules/@dreipol/storybook-figma-addon/node_modules
        using description file: /Users/borgca/Repo/fox/node_modules/@dreipol/storybook-figma-addon/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
      looking for modules in /Users/borgca/Repo/fox/node_modules
        using description file: /Users/borgca/Repo/fox/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
          using description file: /Users/borgca/Repo/fox/node_modules/storybook-addon-designs/package.json (relative path: ./lib/register/components/Image)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
          using description file: /Users/borgca/Repo/fox/node_modules/@dreipol/storybook-figma-addon/package.json (relative path: ./node_modules/storybook-addon-designs/lib/register/components/Image)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/borgca/Repo/fox/node_modules/storybook-addon-designs/lib/register/components/Image doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              /Users/borgca/Repo/fox/node_modules/@dreipol/storybook-figma-addon/node_modules/storybook-addon-designs/lib/register/components/Image doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              /Users/borgca/Repo/fox/node_modules/storybook-addon-designs/lib/register/components/Image.mjs doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/borgca/Repo/fox/node_modules/@dreipol/storybook-figma-addon/node_modules/storybook-addon-designs/lib/register/components/Image.mjs doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/borgca/Repo/fox/node_modules/storybook-addon-designs/lib/register/components/Image.js doesn't exist
            .jsx
              Field 'browser' doesn't contain a valid alias configuration
              /Users/borgca/Repo/fox/node_modules/@dreipol/storybook-figma-addon/node_modules/storybook-addon-designs/lib/register/components/Image.js doesn't exist
            .jsx
              Field 'browser' doesn't contain a valid alias configuration
              /Users/borgca/Repo/fox/node_modules/storybook-addon-designs/lib/register/components/Image.jsx doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration
              /Users/borgca/Repo/fox/node_modules/@dreipol/storybook-figma-addon/node_modules/storybook-addon-designs/lib/register/components/Image.jsx doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration
              /Users/borgca/Repo/fox/node_modules/storybook-addon-designs/lib/register/components/Image.json doesn't exist
            as directory
              /Users/borgca/Repo/fox/node_modules/storybook-addon-designs/lib/register/components/Image doesn't exist
              /Users/borgca/Repo/fox/node_modules/@dreipol/storybook-figma-addon/node_modules/storybook-addon-designs/lib/register/components/Image.json doesn't exist
            as directory
              /Users/borgca/Repo/fox/node_modules/@dreipol/storybook-figma-addon/node_modules/storybook-addon-designs/lib/register/components/Image doesn't exist

WARN FATAL broken build!, will close the process,
WARN Fix the error below and restart storybook.

error Command failed with exit code 1.
```

--- a/package.json
+++ b/package.json
@@ -35,11 +35,11 @@
         "@types/webpack-env": "^1.13.9",
         "lodash": "^4.17.11",
         "react": "^16.8.4",
-        "storybook-addon-designs": "^5.1.0",
+        "storybook-addon-designs": "^5.2.1",
         "typescript": "^3.4.5"
     },
     "peerDependencies": {
-        "storybook-addon-designs": "^5.1.0",
+        "storybook-addon-designs": "^5.2.1",
         "react": ">=16.8.0"
     }
 }

--- a/src/panel.tsx
+++ b/src/panel.tsx
@@ -1,10 +1,10 @@
 /* eslint-disable @typescript-eslint/ban-ts-ignore */
 import React, { Fragment, ReactElement, useEffect, useState } from 'react';
 import { STORY_CHANGED, STORY_RENDERED } from '@storybook/core-events';
-import { ImageConfig } from 'storybook-addon-designs/lib/config';
+import { ImageConfig } from 'storybook-addon-designs/esm/config';
 // @ts-ignore
 import { Placeholder, TabsState } from '@storybook/components';
-import ImagePreview from 'storybook-addon-designs/lib/register/components/Image';
+import ImagePreview from 'storybook-addon-designs/esm/register/components/Image';
 import { loadFigmaImagesByIDs } from './util';
 
 import { Channel } from '@storybook/channels';

